### ISSUE TITLE
6.5.1 ideas

### ISSUE THREAD

**Issue by foxsouns** [BinBashBanana/webretro#83]
@BinBashBanana: i think we should ignore 6.6 goals (offline/mobile support) and just get some bug/breakfixes out. i know you've said you had the ability to fix fast-forward and reduce loading times (due to broken comp flags, iirc?); and sega cd cores were an easy fix to add in. i really don't feel like learning how to screw around with browser storage to bring 6.6 to fruition. if you want, you can salvage what there is from my fork; including a bit of un-cdn-ing things, and string-based version checking. sorry for the lack of motivation, i've been in a rough spot.

**Comment by BinBashBanana** [comment]
Yes, I think you're right. I've been working on a lot of other projects with mercury lately, and I'm just now getting back to webretro development. I think I have too many plans for this next update, so it's going to be split into 2 or 3 updates. Mobile support is 99% done - I just have to fix a couple bugs. A few other bugs are already fixed on my end (fast forward, state save/load times) and a few new features here and there.
